src/components/Button.tsx

```diff
@@ -0,0 +1,12 @@
+export function Button() {
```

Comment:
Let's ES6+ syntax for arrow functions here like so;

```typescript
const Button = () => { ...your code here }
```

and export your component with the line below at the bottom of your file.

```typescript
export default Button
```